modules/radsBidAdapter.md

```diff
@@ -0,0 +1,36 @@
+# Overview
+
+```
+Module Name: RADS Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: prebid@recognified.net
+```
+
+# Description
+
+RADS Bidder Adapter for Prebid.js 1.x
+
+# Test Parameters
+```
+    var adUnits = [
+        {
+            code: "test-div",
+            mediaTypes: {
+                banner: {
+                    sizes: [[320, 50]]
+                }
+            },
+            bids: [
+                {
+                    bidder: "rads",
+                    params: {
```

Comment:
This ad unit only works with your dev endpoint, so please add `devMode: true` to the params.
Using these test params, the default endpoint returns: `{"error":"Publication not defined."}`

Reply:
@onlsol Changes look good. Thanks